### PR TITLE
Ensure the two copies of citus.control are kept in sync

### PR DESCRIPTION
**Pull request by lithp**
1. Leave a comment reminding they must be kept in sync
2. Add a check to travis which diffs the two files and confirms they're identical

### CODE REVIEW

File: citus.control [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Citus extension
-comment = 'Citus distributed database'
+omment = 'Citus distributed database'
```
Comment (lithp) [inline citus.control]: Added this to show that it fails the build

File: .travis.yml [hunk 2]
```diff
@@ -22,7 +22,7 @@ matrix:
   allow_failures:
     - env: PGVERSION=11
 before_install:
-  - git clone -b v0.7.6 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b add-citus-control-check --depth 1 https://github.com/citusdata/tools.git
```
Comment (lithp) [inline .travis.yml]: I'll update this one https://github.com/citusdata/tools/pull/33 is merged

**Comment by jasonmp85** [comment]
#2111 makes this go away, FWIW 😈…

**Comment by lithp** [comment]
Yes, cmake would be dope

**Comment by jasonmp85** [comment]
Eh, I'm not a fan of this. It's extra complication and bound to be incomplete in other ways later (in fact it already _is_ in the sense that it doesn't check `citus_version.h`, which was also wrong in the offending original PR).

I think we should hold a powwow about Windows support during build time and whether we want to continue the CMake work because without it we're going to keep needing hacks like this.

**Comment by lithp** [comment]
I'm all for fixing the core problem, the fact that we use multiple build systems, but I don't see why that's a reason to block this PR. I'm not introducing any extra complication, just adding some safety to the complication that we already have. And sure, this doesn't solve all problems, but it does solve a problem!

**Comment by jasonmp85** [comment]
Dropping appveyor support (#2541) means we're going with _[dropping] Windows from our build matrix_, so this PR is obsolete.

**Pull request closed without merging**